src/test-fbp/converter-error-expected.txt

```diff
@@ -0,0 +1 @@
+WRN: ./src/lib/common/sol-types.c:242 sol_irange_division() Division by zero: 10, 0
```

Comment:
The node does send an error packet and the test wait for it.

This warning is in our division function `sol_irange_division`.